go.mod

```diff
@@ -1,22 +1,23 @@
 module github.com/meltwater/drone-cache
 
 require (
+	cloud.google.com/go/storage v1.1.0
 	github.com/aws/aws-sdk-go v1.16.35
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/fatih/color v1.7.0 // indirect
 	github.com/go-ini/ini v1.41.0 // indirect
 	github.com/gopherjs/gopherjs v0.0.0-20181103185306-d547d1d9531e // indirect
 	github.com/jtolds/gls v4.2.1+incompatible // indirect
-	github.com/mattn/go-colorable v0.1.4 // indirect
-	github.com/mattn/go-isatty v0.0.10 // indirect
 	github.com/minio/minio-go v6.0.14+incompatible
 	github.com/mitchellh/go-homedir v1.1.0 // indirect
 	github.com/pkg/errors v0.8.1
 	github.com/pkg/sftp v1.10.1
-	github.com/rakyll/gotest v0.0.0-20180125184505-86f0749cd8cc // indirect
 	github.com/smartystreets/assertions v0.0.0-20190116191733-b6c0e53d7304 // indirect
 	github.com/smartystreets/goconvey v0.0.0-20181108003508-044398e4856c // indirect
 	github.com/urfave/cli v1.20.0
 	golang.org/x/crypto v0.0.0-20190820162420-60c769a6c586
+	golang.org/x/sys v0.0.0-20191008105621-543471e840be // indirect
+	google.golang.org/api v0.9.0
 	gopkg.in/ini.v1 v1.41.0 // indirect
 )
+
+go 1.13
```

Comment:
@kakkoyun is it fine to keep this line ? i don't know if this breaks compilation using older versions

Reply:
We should remove it. I'll send another dedicated PR to upgrade everything to 1.13
We have to upgrade container base images, CI images etc.